protocol/src/main/java/org/fido/iot/protocol/Const.java

```diff
@@ -178,6 +178,7 @@
   public static final String VALIDATOR_ALG_NAME = "PKIX";
 
   //EC curve names
+  public static final String PRIME256V1_CURVE_NAME = "prime256v1";
```

Comment:
Could you please help me understand why is the same not done for ECDSA 384 Curve? 
Is it because this is a fix for automation which by default uses only ECDSA 256?

Reply:
It's a legacy thing.  There are two names for the NIST P-256 curve, but only one for the NIST-384 curve.
I assume it's because the 256 curve was defined before being introduced into the SECP lists.

Reply:
ok. Thanks!!